dcos/ingress/linkerd-dcos.json

```diff
@@ -8,7 +8,7 @@
   "container": {
     "type": "DOCKER",
     "docker": {
-      "image": "buoyantio/linkerd:1.0.0",
+      "image": "buoyantio/linkerd:1.0.1",
```

Comment:
I think you also need to update the file path to the linkerd executable on line 41 of this file. Same goes for the rest of the dcos configs.